architecture.md

```diff
@@ -644,9 +644,40 @@ a pod is **not** `Privileged` the runtime selection is done as follows:
 | Default CRI-O trust level: `untrusted` | Kata Containers                               | Kata Containers                              |  Kata Containers |
 
 
-### CRI-containerd
+### Containerd CRI Plugin (Formerly CRI-containerd)
```

Comment:
Alas, github markdown (unlike rST) cannot auto-generate the TOC, so please can you update the table of contents for this new section name?

Reply:
Thanks, @jodh-intel @jcvenegas should I amend the commit with the changes or create a new commit? would like to get a sense if there's a standard practice.

Reply:
We do amend and push again - to keep the semantic in the commit when modifications  correspond the the same change.